apps/preview/nuxt/pages/showcases/Modal/ModalBasic.vue

```diff
@@ -29,7 +29,13 @@
 </template>
 
 <script lang="ts" setup>
+import { ref, watch } from 'vue';
 import { SfModal, SfButton, SfIconClose, useDisclosure } from '@storefront-ui/vue';
-
+import { useFocus } from '@vueuse/core';
+const buttonRef = ref();
+const { focused } = useFocus(buttonRef);
 const { isOpen, open, close } = useDisclosure({ initialValue: false });
```

Comment:
```suggestion
const { isOpen, open, close } = useDisclosure();
```
default is initialValue: false

Reply:
removed